src/Amazon.DynamoDb.Tests/Requests/UpdateItemTests.cs

```diff
@@ -13,28 +14,28 @@ public void UpdateItemRequest1()
 		{
             var date = DateTimeOffset.FromUnixTimeSeconds(1497282355);
 
-            var request = new UpdateItemRequest("Entities", Key<Entity>.FromValues(1), new[] {
+            var request = new UpdateItemRequest("Entities", Key<Entity>.FromValues(1).ToDictionary(x => x.Key, y => new DbValue(y.Value)), new[] {

```

Comment:
I put this in to get it to compile. I thought about adding a custom JsonConverter for `Key`, but the problem there is that `UpdateItemRequest` was using `IEnumerable<string, object>`, so there's no guarantee that it's a `Key` instance. Maybe we switch to `IDictionary<string, object>()` so System.Text.Json knows how to serialize it? I didn't want to get into making any proposed changes to `Carbon.Data` though